static/tests/RouterNotFound.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import RouteNotFound from '../components/RouteNotFound.jsx';
+
+
+test('RouteNotFound Snapshot', () => {
+  const component = renderer.create(
+    <RouteNotFound />
+  );
+  let tree = component.toJSON();
+  expect(tree).toMatchSnapshot();
+})
```

Comment:
So, `RouteNotFound` is basically a completely static component.

I don't think it's worthwhile to test these. Basically, if you ever update this file, the snapshot test will say "hey, you changed this – did you expect to?". And of course you did – it's the only possible thing you could have meant (since there's no behavior to change).

I'd remove this test, and any other snapshot tests of pure static components.